tools/jira_cmd.py

```diff
@@ -53,11 +53,11 @@ def log_exception(msg):
         logger.error(msg)
 
 
-def jira_netrc_login(netrcFile):
+def jira_netrc_login(server, netrcFile):
```

Comment:
Why do you need this change?

Reply:
my password contains a space
so netrc cannot contain it :(